frontend/src/pages/operation_list/index.tsx

```diff
@@ -26,9 +27,63 @@ export default () => {
   ))
   const filterText = useFormField<string>('')
 
+  const [ops, setOps] = React.useState<Operation[]>([])
+  const [welcomeFlag, setWelcomeFlag] = React.useState<boolean>(false)
+
+  React.useEffect(() => {
+    wiredData.expose(data => {
+      if (data) {
+        const [ops, welcomeFlag] = data
+        setOps(ops)
+        setWelcomeFlag(welcomeFlag)
+      }
+    })
+  }, [wiredData])
+
+  type Header = "Other" | "Favorites" | null
+  const showOperationList = (ops: Operation[], header: Header) => (
```

Comment:
This works, but I'd consider moving this into a stand alone component.

Reply:
for sure!

Reply:
changed this into a component

Reply:
actually fixing a bug rn that I created when I made this change

Reply:
okay fixed it